app/backend/lib/headers.ts

```diff
@@ -1,24 +1,22 @@
-import helmet from "helmet";
-import config from "../../config";
+import helmet from 'helmet';
+import config from '../../config';
 
 const headersMiddleware = () => {
   const helmetMiddleware = helmet({
+    // CSP header is set in app/middleware.ts, to be compatible with next.js
     contentSecurityPolicy: false,
   });
   return (req, res, next) => {
     // Tell search + crawlers not to index non-production environments:
     if (
-      !config.get("OPENSHIFT_APP_NAMESPACE") ||
-      !config.get("OPENSHIFT_APP_NAMESPACE").endsWith("-prod")
+      !config.get('OPENSHIFT_APP_NAMESPACE') ||
+      !config.get('OPENSHIFT_APP_NAMESPACE').endsWith('-prod')
     ) {
-      res.append("X-Robots-Tag", "noindex, noimageindex, nofollow, noarchive");
+      res.append('X-Robots-Tag', 'noindex, noimageindex, nofollow, noarchive');
     }
 
-    // https://www.zaproxy.org/docs/alerts/10038/
```

Comment:
app\tests\backend\headers.test.ts needs to be updated to remove assertion on `res.append("Content-Security-Policy" ..`